hardhat_scripts/staging_redeem_queue_migration/redeem_queue_repair.ts

```
@@ -0,0 +1,418 @@
+    "function repairRedeemQueue((uint256 amount,uint256 maxRedeemableEth,address recipient,uint256 height,address initiator)[] _requests,bytes32 _expectedQueueHash)",
+  ]);
+  return iface.encodeFunctionData("repairRedeemQueue", [
+    rows.map((r) => [r.amount, r.maxRedeemableEth, r.recipient, r.height, r.initiator]),
+    expectedQueueHash,
+  ]);
+}
+
+export function eth(v: EthersType.BigNumber): string {
+  const s = EthersType.utils.formatEther(v);
+  const [i, f = ""] = s.split(".");
+  return `${BigInt(i).toLocaleString("en-US")}.${(f + "000000").slice(0, 6)}`;
```

ESLint

[error] The 'BigInt' is not supported until Node.js 10.4.0. The configured version range is '>=8.0.0'.

(node/no-unsupported-features/es-builtins)